packages/iTwinUI-react/src/core/Table/actionHandlers/selectHandler.ts

```diff
@@ -56,6 +56,20 @@ export const onSelectHandler = <T extends Record<string, unknown>>(
   onSelect?.(selectedData, newState);
 };
 
+export const onToggleHandler = <T extends Record<string, unknown>>(
+  newState: TableState<T>,
+  action: ActionType,
+  instance?: TableInstance<T>,
+  onSelect?: (
+    selectedData: T[] | undefined,
+    tableState?: TableState<T>,
+  ) => void,
+  isRowDisabled?: (rowData: T) => boolean,
+) => {
+  onSelectHandler(newState, instance, onSelect, isRowDisabled);
+  newState.lastSelectedRowId = action.id;
```

Comment:
This line is very cryptic so a comment would be useful here.

Also we might want to document somewhere what is the expected flow for the various permutations of multi-selection. Because I would never have guessed that clicking on a checkbox should update "start of shift selection".

Reply:
> Not sure, but is this what you meant?
 
That's not super helpful, I think it should explain *why* this line is needed or what value it adds, rather than simply what it does.

> Would the documentation website be the right place for this? Or are you suggesting somewhere in the code/JSDocs?

Documentation website might work yeah, but I expect the Table page to get very long, so I'm not sure if that's the only place we should mention it.

Reply:
> That's not super helpful, I think it should explain why this line is needed or what value it adds, rather than simply what it does.

Added comment on [L71](https://github.com/iTwin/iTwinUI-react/pull/889/files#diff-772f25aaf3cd0aa17d5900a4252050c32c48b2efe18b4f6db0d84df08f96cc0bR71-R72)

Reply:
> Documentation website might work yeah, but I expect the Table page to get very long, so I'm not sure if that's the only place we should mention it.

I tried adding comments to help explain the expected workflow of ctrl/shift selection.

* In this PR, I added some JSDocs for some of the methods in `selectHandler.ts`. [Commits](https://github.com/iTwin/iTwinUI-react/pull/889/files/a52301ad0889ebd3794ee73c6d1d1242de587607..0eaab948add2cd1535f078c1b2dd5d5e270515d7)
* In the related PR [#888](https://github.com/iTwin/iTwinUI-react/pull/888), I added a small comment talking about previous selection handling depending on if it is a shift click or a ctrl + shift click. [Commit](https://github.com/iTwin/iTwinUI-react/pull/888/commits/f37c20902961eb76118aa148f7a0d21f467688c2)

I will also add it in some place in the documentation website soon.